.rubocop.yml

```
@@ -0,0 +1,26 @@
+# The behavior of RuboCop can be controlled via the .rubocop.yml
+# configuration file. It makes it possible to enable/disable
+# certain cops (checks) and to alter their behavior if they accept
+# any parameters. The file can be placed either in your home
+# directory or in some project directory.
+#
+# RuboCop will start looking for the configuration file in the directory
+# where the inspected file is and continue its way up to the root directory.
+#
+# See https://docs.rubocop.org/rubocop/configuration
+---
+inherit_from: .rubocop_todo.yml
+
+require:
+- rubocop-performance
+- rubocop-rake
+- rubocop-rspec
+
+AllCops:
+  NewCops: enable
+  DisplayCopNames: true
+  ExtraDetails: true
+  DisplayStyleGuide: true
+  TargetRubyVersion: '2.5'
```

at the moment the gem supports 2.4 (we should add that to the gemspec).
```suggestion
  TargetRubyVersion: '2.4'
```

---

Does it mean I should change the ruby version to 2.4 too then?

---

Yes.

---

Tbh, I'd prefer to run latest rubocop on ruby 2.7 with `TargetRubyVersion: '2.4'` these days.. 🤔

---

I think we can also move to Ruby 2.7+ now. Perhaps something we should do across all Beaker gems?

---

So far I'll use ruby 2.7 in GHA but I'll target ruby 2.4 in rubocop. We can bump dependencies later then